check_license.sh

```diff
@@ -61,10 +61,8 @@ sed '/^$/d' $CHKSUM_FILE > $TMP_CHKSUM_FILE
 # Use the tmp-file for the rest of the script
 CHKSUM_FILE=$TMP_CHKSUM_FILE
 
-# Collect only stderr from the subcommand
 output="$(
-          exec 3>&1
-          shasum --warn --algorithm 256 --check $CHKSUM_FILE > /dev/null 2>&3
+          shasum --warn --algorithm 256 --check $CHKSUM_FILE
```

Comment:
Hmm, is this right? Previously stderr was captured by the `$()` because of the `exec 3>&1` and `2>&3` (not sure why the `exec` is needed though), but now it is not redirected anywhere and escapes the capture. This means that the `grep` below will probably never trigger.

Reply:
Good catch @kacf !

Reply:
Thanks @kacf , for now I'll convert this PR to draft so no one merges it